config.json

```diff
@@ -485,6 +485,19 @@
           "loops",
           "math"
         ]
+      },
```

Comment:
Can we please move this earlier (I'd suggest just after leap) to facilitate recommended order on the website?

Reply:
Changed as requested